engines/basic_resources/app/controllers/basic_resources/api/v1/memberships_controller.rb

```diff
@@ -61,8 +60,12 @@ def destroy
 
         private
 
+        def group
+          Group.find(params[:group_id]) if params.key?(:group_id)
```

Comment:
I agree, it will be troublesome for the client not to be consistent in the naming. Another thing is aliasing that column name in the response as soon as we find a better naming. All the encapsulation we can implement, we'll ease further changes.